packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -64,25 +58,27 @@ const { usePostActions } = unlock( editorPrivateApis );
 const { useLocation } = unlock( routerPrivateApis );
 
 const EMPTY_ARRAY = [];
-const defaultConfigPerViewType = {
-	[ LAYOUT_TABLE ]: {
-		primaryField: 'title',
-	},
-	[ LAYOUT_GRID ]: {
-		mediaField: 'preview',
-		primaryField: 'title',
-		badgeFields: [ 'sync-status' ],
-	},
+const defaultFields = {
+	[ LAYOUT_TABLE ]: [
+		'preview',
+		{ render: 'primary', field: 'title' },
```

Comment:
```js
{
  fields: [
    { field: 'title', render: 'primary' }
  ]
}
```

Can we use `id` instead of `field`? It seems a bit inconsistent (different from the field config) and redundant (each item in the list is a field):

```js
{
  fields: [
    { id: 'title', render: 'primary' }
  ]
}
```

Reply:
We can but I believe we use field to refer to fields elsewhere in the view object like in filters, order and we should be consistent with all the places in the view object

Reply:
I see, that's a fair point 👍